src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/TickerTask.java

```diff
@@ -416,9 +419,27 @@ public boolean isHalted() {
 
     /**
      * This method orders the {@link TickerTask} to be halted.
+     * Once halted the task <strong>cannot</strong> be un-halted.
+     * This method will block until all sync tasks are processed; however,
+     * it will not block for the async tasks
      */
     public synchronized void halt() {
+        /*
```

Comment:
```suggestion
    public synchronized void halt() {
        SlimefunPlugin.logger().info("Stopping TickerTask, this may take a few seconds or up to a minute...");
        /*
```